common/buildcraft/transport/plug/PluggableGate.java

```diff
@@ -175,7 +176,14 @@ public void sendGuiMessage(int id, IPayloadWriter writer) {
     }
 
     @Override
-    public void readPayload(PacketBuffer buffer, Side side, MessageContext ctx) throws IOException {
+    public void writePayload(PacketBuffer buffer, Side side) {
+        // What???
```

Comment:
what are you doing here indeed?

Reply:
Packet sending is odd with pluggables. Its something I need to look at a bit more...